element_calcium_imaging/imaging_no_curation.py

```diff
@@ -522,16 +524,33 @@ def make(self, key):
             else:
                 raise NotImplementedError("Unknown method: {}".format(method))
         elif task_mode == "trigger":
+            drop_frames = (ZDriftMetrics & key).fetch1("bad_frames")
```

Comment:
Because there may be multiple entries in `ZDriftMetrics` for one particular scan (corresponding to different zdrift_paramset), there is a risk with `fetch1()` here.
There is no guarantee that there is only one. If there are multiple `ZDriftMetrics` for this scan, which zdrift_param are we going with? Can we make the assumption to go with the first one?

Reply:
That is a good point. We can probably assume the first one for now, unless we want to refactor the module to allow `zdrift_paramset_idx` as an additional primary key for all downstream tables from `ZDriftMetrics`

Reply:
Since this is a fork separate from the main element, I'm open to either approach. Let me know which one you prefer

Reply:
@kushalbakshi , let's go with the assumption that there is only one `ZDriftMetrics` for the particular scan, raise an error with a meaningful message if there are more than one.